.github/workflows/build.yaml

```
@@ -65,22 +63,34 @@ jobs:
               - 'dataplane/**'
             containerfile:
               - 'build/Containerfile.dataplane'
-      - name: Setup QEMU
-        if: steps.filter.outputs.dataplane || steps.filter.outputs.containerfile
-        uses: docker/setup-qemu-action@29109295f81e9208d7d86ff1c6c12d2833863392 # v3.6.0
-      - name: Setup Docker Buildx
+      - name: Build container image for amd64
         if: steps.filter.outputs.dataplane || steps.filter.outputs.containerfile
-        id: buildx
-        uses: docker/setup-buildx-action@b5ca514318bd6ebac0fb2aedd5d36ec1b5c232a2 # v3.10.0
+        uses: docker/build-push-action@5cd11c3a4ced054e52742c5fd54dca954e0edd85 # v6.7.0
         with:
-          buildkitd-flags: "--debug"
-      - name: Build multi-arch container image
+          push: false
+          context: .
+          file: build/Containerfile.dataplane
+          tags: kong/blixt-dataplane:pr-${{ github.event.pull_request.number }}-amd64-${{ github.sha }}
+
+  docker-build-arm64:
+    name: docker-build-arm64
+    runs-on: ubuntu-22.04-arm
+    steps:
+      - name: Checkout
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+      - uses: dorny/paths-filter@v3
+        id: filter
+        with:
+          filters: |
+            dataplane:
+              - 'dataplane/**'
+            containerfile:
+              - 'build/Containerfile.dataplane'
+      - name: Build container image for arm64
         if: steps.filter.outputs.dataplane || steps.filter.outputs.containerfile
         uses: docker/build-push-action@471d1dc4e07e5cdedd4c2171150001c434f0b7a4 # v6.15.0
         with:
           push: false
-          builder: ${{ steps.buildx.outputs.name }}
           context: .
-          platforms: linux/amd64,linux/arm64
           file: build/Containerfile.dataplane
-          tags: kong/blixt-dataplane:pr-${{ github.event.pull_request.number }}-${{ github.sha }}
+          tags: kong/blixt-dataplane:pr-${{ github.event.pull_request.number }}-arm64-${{ github.sha }}
```

```suggestion
          tags: localhost/blixt-dataplane:pr-${{ github.event.pull_request.number }}-arm64-${{ github.sha }}
```

(again, this can change later when #339 is completed)